sktime/param_est/seasonality.py

```diff
@@ -407,3 +407,130 @@ def get_test_params(cls, parameter_set="default"):
         params2 = {"candidate_sp": [3, 7, 12]}
 
         return [params1, params2]
+
+
+class SeasonalityPeriodogram(BaseParamFitter):
+    """Score periodicities by their spectral power.
+
+    Interfacing `seasonal.periodogram` to determine candidate seasonality parameters.
+
+    Parameters
+    ----------
+    candidate_sp : None, int or list of int, optional, default = None
```

Comment:
from the code, it looks like the default is 4?

Further, I think the mapping of `candidate_sp` on `min_period` and `max_period` might be confusing to the user.
Perhaps you just want to patch through `min_period` and `max_period`, that might be clearer?